docs/design/registries.md

```diff
@@ -327,23 +319,25 @@ version = "1.2.3"
 # Rust crate dependencies here
 
 [package.metadata.component]
-package = "my-org/my-component"
+package = "my-org:my-component"
 
 [package.metadata.component.target]
 path = "wit"
 
 [package.metadata.component.target.dependencies]
-wasi = "webassembly/wasi@1.2.3"
+"wasi:cli" = "1.2.3"
 
 [package.metadata.component.dependencies]
-regex = "fancy-components/regex@1.0.0"
-transcoder = "fancy-components/transcoder@1.0.0"
+"fancy-components:regex" = "1.0.0"
+"fancy-components:transcoder" = "1.0.0"
 ```
 An example `wit/component.wit`:
 
 ```wit
-default world my-component {
-  include wasi.command # a theoretical syntax for including a world in another
+package my-org:my-component
+
+world my-component {
```

Comment:
Similarly here, to help people understand the role of the package name, could you rename one of the `my-component` names so that it's easy to tell what's influenced by what?

Reply:
Similarly, I'll update this too.